juju/juju
# Feature/add device2state

## Description of change

*Please replace with a description about why this change is needed?*

Write `Device` charm metadata into mongo from `state`.

## QA steps

*Please replace with how we can verify that the change works?*

This is testable after operator supports `device` in the future.

## Documentation changes

*Please replace with any notes about how it affects current user workflow? CLI? API?* 

nil
